README.md

```diff
@@ -317,3 +379,37 @@ which either had errors or already existed. Note that a
 successful response from this method does not mean that _all_ of
 the features were created (since, for example, some may already
 exist), only that _some_ of them were.
+
+### Class: `Answer`
+
+`Answer` is the type of object returned by `tier.can()`.
+
+#### `answer.ok`
+
+`ok` reports if the program should proceed with a user request or
+not. To prevent total failure if `can()` needed to reach the sidecar
+and was unable to, `ok` will fail optimistically and report true.
+If the opposite is desired, clients can check `err`.
+
+#### `answer.err`
+
+Any error encountered fetching the `Usage` record for the org and
+feature.
+
+#### `answer.report([n = 1])`
+
+Report the usage in the amount specified, default `1`.
+
+#### `answer.limit`
+
+Number specifying the limit for the feature usage.
+
+#### `answer.used`
+
+Number specifying the amount of the feature that the org has
+consumed.
+
+#### `answer.remaining`
```

Comment:
Let's hold off introducing this for now. At the moment, the Go SDK does use the LookupLimits API, which returns some (stale) usage along with a limit; however, it decisively does not leak that information so that later, when the time comes, the sidecar can begin caching and ignoring limits under any circumstance it deems okay to do so. That is to say, the sidecar only guarantees soft limits and is sometimes not based on usage.